nixos/modules/services/web-apps/mastodon.nix

```
@@ -90,6 +90,10 @@ let
     SystemCallArchitectures = "native";
   };
 
+  commonServices = lib.optional redisActuallyCreateLocally "redis-mastodon.service"
```

```suggestion
  # Services that all Mastodon units After= and Requires= on
  commonServices = lib.optional redisActuallyCreateLocally "redis-mastodon.service"
```

Can we have an explaining comment for this?

---

Ok, update PR.